cni.go

```diff
@@ -41,10 +41,12 @@ type CNI interface {
 	Check(ctx context.Context, id string, path string, opts ...NamespaceOpts) error
 	// Load loads the cni network config
 	Load(opts ...Opt) error
-	// Status checks the status of the cni initialization
-	Status() error
+	// Status executes the status verb of the cni plugin
+	Status(ctx context.Context) error
```

Comment:
great, it already existed the concept

Reply:
Yes. Status detected if the minimum number of networks was met. In containerd cri-plugin, its hard coded to 2 neworks. Loopback and Default CNI network configuration. I changed the existing to 'Ready' and have Status calling the status function out of libcni.